applications/plugins/LMConstraint/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+cmake_minimum_required(VERSION 3.12)
+project(LMConstraint VERSION 1.0 LANGUAGES CXX)
+
+sofa_find_package(SofaFramework REQUIRED)
+sofa_find_package(SofaBase REQUIRED)
+sofa_find_package(SofaCommon REQUIRED) 
```

Comment:
(Github did not save this comment while doing the previous review 👹)
SofaCommon is deprecated, so use the newly pluginization process and use find_package(SofaImplicitOdeSolver  REQUIRED) 😚